From: Johannes Thumshirn <jthumshirn@suse.de>
Date: Thu Dec 21 14:07:56 CET 2017
Subject: Fix unsed variable warning in has_unmovable_pages
References: bsc#1073868
Patch-mainline: Never, SUSE back-port specific

GCC complains about the following:
  mm/page_alloc.c: In function ‘has_unmovable_pages’:
  mm/page_alloc.c:7005:6: warning: unused variable ‘mt’ [-Wunused-variable]
    int mt;
        ^

Which is likely caused by:
'patches.fixes/mm-distinguish-CMA-and-MOVABLE-isolation-in-has_unmo.patch'

Get rid of 'int mt' as well.

Signed-off-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 mm/page_alloc.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/page_alloc.c
+++ b/mm/page_alloc.c
@@ -6929,7 +6929,6 @@ bool has_unmovable_pages(struct zone *zo
 			 bool skip_hwpoisoned_pages)
 {
 	unsigned long pfn, iter, found;
-	int mt;
 
 	/*
 	 * For avoiding noise data, lru_add_drain_all() should be called
